src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -156,82 +156,81 @@ <h2 class="text-muted">
-          </strong>
-        </button>
-      </span>
-      <span class="float-sm-right">
-        <button class="btn btn-light btn-sm" *ngIf="canRestoreAll" (click)="onRestoreAll()"
-            ngbTooltip="Restore all deleted courses and their corresponding students and sessions">
-          <strong class="text-color-black">
-            <span class="fa fa-check-circle"></span> Restore All
-          </strong>
-        </button>
-        <button class="btn btn-light btn-sm disabled" *ngIf="!canRestoreAll">
-          <strong class="text-color-black">
-            <span class="fa fa-check-circle"></span> Restore All
-          </strong>
-        </button>
-      </span>
-    </div>
-  </div>
-  <div class="table-responsive">
-    <table class="table table-striped table-bordered margin-0">
-      <thead>
-        <tr class="background-color-medium-gray text-color-gray">
-          <th>Course ID</th>
-          <th>Course Name</th>
-          <th>Creation Date</th>
-          <th>Deletion Date</th>
-          <th class="text-center">Action(s)</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr *ngFor="let course of softDeletedCourses">
-          <td>{{course.id}}</td>
-          <td>{{course.name}}</td>
-          <td container="body" [ngbTooltip]="course.createdAt | date:'EEE, dd MMM yyyy, h:mma'">
-            {{course.createdAt | date:'d MMM yyyy'}}
-          </td>
-          <td container="body" [ngbTooltip]="course.deletedAt | date:'EEE, dd MMM yyyy, h:mma'">
-            {{course.deletedAt | date:'d MMM yyyy'}}
-          </td>
-          <td class="text-center">
-            <button class="btn btn-light btn-sm" *ngIf="course.canModifyCourse"
-                (click)="onRestore(course.id)"
-                ngbTooltip="Restore the deleted course and its corresponding students and sessions">
-              Restore
-            </button>
-            <button class="btn btn-light btn-sm disabled" *ngIf="!course.canModifyCourse">
-              Restore
-            </button>
-            <button class="btn btn-light btn-sm" *ngIf="course.canModifyCourse" style="color: red"
-                (click)="onDeletePermanently(course.id)"
-                ngbTooltip="Permanently delete the course and its corresponding students and sessions">
-              Delete Permanently
+<div class="row course-section margin-top-30px" *ngIf="softDeletedCourses.length">
+  <div class="col-12">
+   <h2 class="text-muted">
+      <span class="fa fa-trash-alt"></span> Deleted Courses
+    </h2>
+    <div class="card bg-light top-padded">
+      <div class="card-header bg-secondary recycle-bin-header" (click)="isRecycleBinExpanded = !isRecycleBinExpanded">
+        <div class="row align-items-center">
+          <div class="col-6 text-white">
+            <b>Recycle Bin</b>
+          </div>
+          <div class="col-6 text-right">
+            <button class="btn btn-light btn-sm no-margin" *ngIf="canRestoreAll" (click)="onRestoreAll()"
```

Comment:
Right now, clicking this button toggles `isRecycleBinExpanded`, which shouldn't be the case
(it should only be toggled when we click anywhere else at the header, but not the button)

_Hint: see how it's being done in the feedback session's recycle bin :)_

Reply:
It is supposed to worked as you explained. The click event code is at a container level.
https://github.com/TEAMMATES/teammates/blob/f12975ccee4a01154ec81a94d657d517009e17b2/src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html#L165
![ezgif-1-fcb7da3b9cac](https://user-images.githubusercontent.com/20428290/55596066-d40b8d00-5714-11e9-829c-0b6b5196f417.gif)

Reply:
Yup, it works fine when you click the grey header, but try clicking the `Delete All` button. 
A modal will show up, but at the same time, the recycle bin expands/collapses at the background. We don't want it to expand/collapse when we click the `Restore All` or `Delete All` button.
Meanwhile, the feedback session's recycle bin doesn't show this behavior.

Reply:
Got it!